git2.nobj.lua

```diff
@@ -1,5 +1,7 @@
 
 basetype "git_time_t"  "integer" "0"
+-- really? is this it?
```

Comment:
Yes.  That is how to handle enums for now.

In the future it would most likely be better for LuaNativeObjects to create a custom UserData type for each enum type and wrap the enum variants in that type.  This would allow parameter type checking, but will require a lot of work to add support.